src/test/java/com/practice/test/CalculatorServiceTest.java

```diff
@@ -48,5 +48,6 @@ public void divisionReturnPositiveWhenDivisorAndDividendValueArePositive(){
     @Test
     public void multiplyOperation(){
```

Comment:
Test case name is not descriptive enough, please provide a more descriptive naming convention example: muliplyfunctionIsSuccessfullWhenTwoNumberAreMulitplied()

Reply:
Hi, thank you for the feedback. I have done the change as per your suggestion. Please review. Thank you.